src/main/java/org/folio/linked/data/integration/ResourceModificationEventListener.java

```diff
@@ -42,6 +43,13 @@ public void afterCreate(ResourceCreatedEvent resourceCreatedEvent) {
     sendToInventory(resource);
   }
 
+  @TransactionalEventListener
+  public void afterCreate(ResourceAuthorityCreatedEvent resourceCreatedEvent) {
```

Comment:
`AuthorityResourceCreatedEvent` may be a better name.

Also, we will have to rename the existing `ResourceCreatedEvent` to `BibliographicResourceCreatedEvent`
(because "Authority" and "Bibliographic" are sibling terms). 

The term "Bibliographic" includes  both "Instance" and "Work"
The term "Bibframe" includes "Instance", "Work" and "Authorities".

https://www.loc.gov/marc/bibliographic/
https://www.loc.gov/marc/authority/

Reply:
not actual, remove
only 1 create event exist now